synapse/storage/databases/main/event_federation.py

```diff
@@ -71,6 +72,12 @@
 
 logger = logging.getLogger(__name__)
 
+BACKFILL_EVENT_BACKOFF_UPPER_BOUND_SECONDS: int = int(
+    datetime.timedelta(days=7).total_seconds()
+)
+BACKFILL_EVENT_EXPONENTIAL_BACKOFF_STEP_SECONDS: int = int(
+    datetime.timedelta(hours=1).total_seconds()
+)
```

Comment:
New config?

I'm happy to leave it as constants for now until someone has a need to customize.